### DISCUSSION TITLE
Nuxt.js internal server error when navigating from direct link

### DISCUSSION THREAD

**Discussion by kuji2336** [nuxt/nuxt#9485] [Questions (Nuxt 2)]
I have some problems when I use nuxt.js with SSR, my web application works perfect in development mode, but in production mode, I have some unexpected errors, Errors occur when pages are linked directly, for SSR i am using nuxt asyncdata, And I get the following kind of error. My application runs on domain https://www.example.com/ and Backend API is https://www.example.com/api/book.I can not figure out  what is happening, where is the problem.

I have two questions

Could the reason be the difference between the domains?
2)How can I avoid page crashes on Internal server error if I am using nuxt Axios module?

https://i.stack.imgur.com/cKtlF.png

**Comment by kissu** [comment]
Hey, I cannot find your Stackoverflow question again, not sure if you deleted it or if it's just me.
Have you tried the few ideas that I suggested you? 😄

**Reply by east119** [reply]
I have the same problem
step1: https://kxxs.top
step2: click '《第九关》（校对版全本）作者：小刀锋利' 
step3: Enter the correct page，URL is https://kxxs.top/23/12634
but 
step1: Enter https://kxxs.top/23/12634
Nuxt.js internal server error when navigating from direct link

**Comment by east119** [comment]
I have the same problem
step1: https://kxxs.top
step2: click '《第九关》（校对版全本）作者：小刀锋利' 
step3: Enter the correct page，URL is https://kxxs.top/23/12634
but 
step1: Enter https://kxxs.top/23/12634
Nuxt.js internal server error when navigating from direct link
